src/base/emu-i386/simx86/interp.c

```diff
@@ -2320,13 +2320,12 @@ intop3b:		{ int op = ArOpsFR[D_MO(opc)];
 				case 0: /* SGDT */
 				    /* Store Global Descriptor Table Register */
 				    PC++; PC += ModRM(opc, PC, _mode|DATA16|MSTORE);
-				    error("SGDT not implemented\n");
-				    break;
+				    /* like with UMIP, cpu.c & dpmi.c can take care of this */
+				    goto not_permitted;
```

Comment:
Well, maybe you can add
an UMIP bit for that? Of
course you don't have to
handle the case when its
clear, other than by printing
the old error msg.